app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java

```diff
@@ -23,6 +23,7 @@
 import android.view.MenuItem;
 import android.view.View;
 import android.view.ViewGroup;
+import android.view.WindowManager;
 import android.widget.Button;
```

Comment:
Hi @noahwalugembe , please do not edit LoginActivity at all. This import is irrelevant. Please read your changes line by line on Github diff page as I do. So that you can make sure if your PR is ready or not.

Reply:
Dear @neslihanturanI mead a Mistake   at first by pull requesting from my master branch so when i branch from master i have to remove all the unwanted old code and it is the reason why am facing styling issues. for example to keep on removing import android.view. WindowManager; from LoginActivity.java because it is unwanted in this pull request.

Reply:
@neslihanturan This line {import android.view.WindowManager;} commons-app:master before i am the one who created it in my master their its not supposed to be included in this pull request.

Reply:
Hi @noahwalugembe. Just a tip. To avoid issues like this, better base your PRs on the master of this (destination) repository **always**. For example if [this repository](https://github.com/commons-app/apps-android-commons) has the remote name as `origin` in your local repo then you can use the following command to ensure you always have the base as the master of this repository:

`git checkout -b <new_branch> origin/master`

Reply:
Let me know if you have any doubts in what I said above.

Reply:
> Let me know if you have any doubts about what I said above.

Thanks dear can i  do it now or i can do it in the next contributions.